crates/astria-composer/src/metrics_init.rs

```diff
@@ -73,31 +74,60 @@ pub fn register() {
     );
 }
 
-pub const TRANSACTIONS_RECEIVED: &str = concat!(env!("CARGO_CRATE_NAME"), "_transactions_received");
+declare_metric_const!(pub const TRANSACTIONS_RECEIVED);
+declare_metric_const!(pub const TRANSACTIONS_DROPPED);
+declare_metric_const!(pub const TRANSACTIONS_DROPPED_TOO_LARGE);
+declare_metric_const!(pub const NONCE_FETCH_COUNT);
+declare_metric_const!(pub const NONCE_FETCH_FAILURE_COUNT);
+declare_metric_const!(pub const NONCE_FETCH_LATENCY);
+declare_metric_const!(pub const CURRENT_NONCE);
+declare_metric_const!(pub const SEQUENCER_SUBMISSION_LATENCY);
+declare_metric_const!(pub const SEQUENCER_SUBMISSION_FAILURE_COUNT);
+declare_metric_const!(pub const TRANSACTIONS_PER_SUBMISSION);
+declare_metric_const!(pub const BYTES_PER_SUBMISSION);
 
-pub const TRANSACTIONS_DROPPED: &str = concat!(env!("CARGO_CRATE_NAME"), "_transactions_dropped");
-
-pub const TRANSACTIONS_DROPPED_TOO_LARGE: &str =
-    concat!(env!("CARGO_CRATE_NAME"), "_transactions_dropped_too_large");
-
-pub const NONCE_FETCH_COUNT: &str = concat!(env!("CARGO_CRATE_NAME"), "_nonce_fetch_count");
-
-pub const NONCE_FETCH_FAILURE_COUNT: &str =
-    concat!(env!("CARGO_CRATE_NAME"), "_nonce_fetch_failure_count");
-
-pub const NONCE_FETCH_LATENCY: &str = concat!(env!("CARGO_CRATE_NAME"), "_nonce_fetch_latency");
-
-pub const CURRENT_NONCE: &str = concat!(env!("CARGO_CRATE_NAME"), "_current_nonce");
-
-pub const SEQUENCER_SUBMISSION_LATENCY: &str =
-    concat!(env!("CARGO_CRATE_NAME"), "_sequencer_submission_latency");
-
-pub const SEQUENCER_SUBMISSION_FAILURE_COUNT: &str = concat!(
-    env!("CARGO_CRATE_NAME"),
-    "_sequencer_submission_failure_count"
-);
+#[cfg(test)]
+mod tests {
+    use super::{
+        BYTES_PER_SUBMISSION,
+        CURRENT_NONCE,
+        NONCE_FETCH_COUNT,
+        NONCE_FETCH_FAILURE_COUNT,
+        NONCE_FETCH_LATENCY,
+        SEQUENCER_SUBMISSION_FAILURE_COUNT,
+        SEQUENCER_SUBMISSION_LATENCY,
+        TRANSACTIONS_DROPPED,
+        TRANSACTIONS_DROPPED_TOO_LARGE,
+        TRANSACTIONS_PER_SUBMISSION,
+        TRANSACTIONS_RECEIVED,
+    };
 
-pub const TRANSACTIONS_PER_SUBMISSION: &str =
-    concat!(env!("CARGO_CRATE_NAME"), "_transaction_per_submission");
```

Comment:
The macro (and test) caught this const being oddly named. This is now `astria_composer_transactions_per_submission`